src/backend/joanie/core/api/client.py

```diff
@@ -29,6 +29,7 @@
 from joanie.core.tasks import generate_zip_archive_task
 from joanie.core.utils import contract as contract_utility
 from joanie.core.utils import contract_definition, issuers
+from joanie.core.utils import query_filter as query_utility
```

Comment:
Why not importing `generate_query_filters` directly ?

Reply:
Finally we use [django-filter](https://pypi.org/project/django-filter/). My method will not exist.